core/orch_test.go

```diff
@@ -765,9 +764,90 @@ func TestPriceInfo_ReturnsBigRat(t *testing.T) {
 	orch := NewOrchestrator(n)
 
 	priceInfo, err := orch.PriceInfo(ethcommon.Address{})
+	assert.Nil(t, err)
+	assert.Equal(t, expPricePerPixel, priceInfo)
+
+	// basePrice = 10/1, txMultiplier = 100/1 => expPricePerPixel = 1010/100
+	basePrice = big.NewRat(10, 1)
+	n.PriceInfo = basePrice
+	orch = NewOrchestrator(n)
+	expPricePerPixel = new(big.Rat).Mul(basePrice, new(big.Rat).Add(big.NewRat(1, 1), new(big.Rat).Inv(txMultiplier)))
```

Comment:
We're repeating this programmatic calculation for `expPricePerPixel` multiple times in this test which is the same logic implemented in the actual `PriceInfo()` method. Perhaps a more robust way to set expectations for these tests is to to hardcode the computed values (which you've already included in comments) for `expPricePerPixel` to ensure that the computation results in the correct value given a set of inputs.

Reply:
Perhaps you're right.
Creating those comments was actually the last thing I did.

Although a human would probably end up calculating those values according to the same formula in `PriceInfo()`, so whether hardcoded or not it doesn't really check the correctness of the formula in the context of creating appropriate price info. Just that the formula returns the results it should return. But I can see the benefit.

Reply:
Human error is certainly possible as well. Though, IMO it feels like we're just duplicating the code snippet from `PriceInfo()` in the test as well so if the code snippet was incorrect the tests would still pass since the expected value is calculated with the same code as the actual value.

Reply:
For reference: 

Hardcoding the values resulted in absolute comparison errors using `Assert.Equal` from Go's testing package. Therefore we are now using `big.Rat.Cmp()` wrapped in `Assert.Zero` from go's testing package, which seems to work.

For some reason the formatting of the denominators is different in the return value from `PriceInfo()` compared to hardcoding values with `big.NewRat()` : `big.nat{} vs big.nat()`

This only happened for two tests though and might be related to a denominator set to `1` even though another test with the same context (denominator of 1) passed without issue. 

```
expected: &big.Rat{a:big.Int{neg:false, abs:big.nat{0x6e}}, b:big.Int{neg:false, abs:big.nat{}}}
actual  : &big.Rat{a:big.Int{neg:false, abs:big.nat{0x6e}}, b:big.Int{neg:false, abs:big.nat(nil)}}
```